po/de.po

```diff
@@ -6,21 +6,55 @@
 # David Auer, 2020
 # Milo Ivir <mail@milotype.de>, 2020, 2021
 # Dr. Tobias Quathamer <toddy@debian.org>, 2021
-# David Kyburz, 2022.
+# David Kyburz, 2022, 2023.
 msgid ""
 msgstr ""
 "Project-Id-Version: pdfarranger\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-09-18 17:02+0200\n"
-"PO-Revision-Date: 2022-09-18 17:17+0200\n"
+"POT-Creation-Date: 2023-01-13 00:51+0100\n"
+"PO-Revision-Date: 2023-01-13 00:59+0100\n"
 "Last-Translator: David Kyburz\n"
 "Language-Team: \n"
 "Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: Poedit 3.1\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#: pdfarranger/config.py:210
+msgid "Preferences"
+msgstr "Einstellungen"
+
+#: pdfarranger/config.py:214 pdfarranger/pdfarranger.py:951
+#: pdfarranger/pdfarranger.py:960 pdfarranger/pdfarranger.py:1061
+#: pdfarranger/pdfarranger.py:1114
+msgid "_Cancel"
+msgstr "_Abbrechen"
+
+#: pdfarranger/config.py:215
+msgid "_OK"
+msgstr ""
+
+#: pdfarranger/config.py:219
+msgid "Language"
+msgstr "Sprache"
+
+#: pdfarranger/config.py:221
+msgid "(Requires restart)"
+msgstr "(Erfordert Neustart)"
+
+#: pdfarranger/config.py:227
+msgid "Theme"
+msgstr "Thema"
+
```

Comment:
I think this is about the color scheme, maybe we can have a look how this is translated and named in other applications.

Reply:
Yes, it's about the color scheme. I will change it to "Erscheinungsbild". This term is used in Firefox. Other ideas are "Stil" (Gnome) or "Modus" (Windows).